pl-csb-model/translate/translator.py

```diff
@@ -4,7 +4,12 @@
 
 def normalize_translation(input: list[str]) -> str:
     # Technically matching against 'Latn' will break if we do languages other than PL and CSB but we won't
-    return re.match(r'^.*Latn (.*)<.*$', input[0]).group(1)
+    exp = r'^.*Latn (.*)<.*$'
+    match = re.match(exp, input[0])
+    # ! Workaround for when model spits out a translation that is too long and doesn't match regex. 
+    # ! Pretend it never happened and return an empty string.
+    # TODO: Add logging to notify us when that happens
```

Comment:
I'll make a separate story for this later. Keeping it as a todo for this pr only.